src/hpv_query.py

```diff
@@ -8,60 +8,71 @@
 
 Without the flag, the script will skip files that have already been processed 
 (but reprocessing the last newsmedia file to ensure no data is incomplete if the process was interrupted mid-file).
-'''
+"""
 
-import pathlib 
-import json
 import argparse
+import json
+import pathlib
+
 from tqdm import tqdm
 
-def hpv_query_terms(): 
-    '''
-    from original paper: 
+def hpv_term_pairs():
```

Comment:
I'm not sure this is necessary unless we expect that requirements will change for querying.
We could probably roll with sth like:
```python
SECOND_TERMS = [...]
def contains_terms(text: str) -> bool:
    text = text.lower()
    res = "hpv" in text
    if not res:
        return False
    # Using a set cause it has O(1) access time
    doc_vocab = set(text.split())
    for term in SECOND_TERMS:
        if term in doc_vocab:
            return True
    return False
```